services/keycloak/Dockerfile

```diff
@@ -88,6 +88,7 @@ COPY entrypoints/default-keycloak-entrypoint.sh /lagoon/entrypoints/99-default-k
 
 COPY startup-scripts /opt/keycloak/startup-scripts
 COPY themes/lagoon /opt/keycloak/themes/lagoon
+COPY providers /opt/keycloak/providers
```

Comment:
I maybe think we should be pulling a versioned copy of this from the source repo release eg
https://github.com/sventorben/keycloak-home-idp-discovery/releases/download/v26.1.1/keycloak-home-idp-discovery.jar (if that's the version added here)

Reply:
They break BC very frequently (v24.0.0, v26.0.0, v26.1.1, etc), and their version numbers don't follow exactly that of keycloak. We can pull it dynamically as long as we don't dependabot update it?

Reply:
yah - we do the same here https://github.com/uselagoon/lagoon/blob/main/services/broker/Dockerfile#L23

Reply:
yes, it will likely need to be updated - but having the versioned plugin documented should make it easier to verify compatibility (same as we do for broker - but broker hard breaks on an incompatible plugin)

Reply:
> This would potentially problematic when we upgrade keycloak versions then?
Yes, we're now blocked on upgrading keycloak until the plugin gets support for that version.

Reply:
It might be worth including a comment in the dockerfile where this is defined
```
FROM quay.io/keycloak/keycloak:26.0.7
```
That says to ensure that a compatible version of this new provider is available before updating keycloak

Reply:
I'll just comment that besides this discussion, from my side everything looks good. At least, it seems to do what it says on the box when I spin it up locally.

Assuming that once the item above is addressed, we're good to go?